src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -4,50 +4,134 @@
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
+
 public class ArrayList<T> implements List<T> {
 
+    private T[] arrayT;
+    private int size;
+
```

Comment:
you should have constant INITAL_CAPACITY and use it while creating your array inside of constructor